### PR TITLE
st1 [1943][MIG][IMP] stock_view_adj_rpl

### PR DESCRIPTION
**Pull request by tack-yabeQuartile**
[#1943](https://www.quartile.co/web#id=1943&action=771&model=project.task&view_type=form&menu_id=505)

* why we migrate this module is solve license conflict. 
* This PR's make "immediate transfer" button (in stock move location wiz.) invisible to prevent accidents.

### CODE REVIEW

**Review by yostashiro — COMMENTED** [review]
We need to switch the license to AGPL.

**Comment by tack-yabeQuartile** [comment]
OK, I'm now fixing license info and update readme and so on.

File: stock_view_adj_rpl/views/stock_move_location_views.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (yostashiro) [inline stock_view_adj_rpl/views/stock_move_location_views.xml]: Please change the file name to `stock_move_location.xml` and move it to `wizard` folder, respecting the structure of the original module.

Comment (tack-yabeQuartile) [reply stock_view_adj_rpl/views/stock_move_location_views.xml]: I got it. I have moved it.

**Review by yostashiro — APPROVED** [review]
LGTM 👍

**Pull request merged**